features/media-regenerate.feature

```diff
@@ -1212,8 +1212,8 @@ Feature: Regenerate WordPress attachments
     And STDERR should be empty
 
   # Audio/video `_cover_hash` meta, used to determine if sub attachment, added in WP 3.9
-  # Test on PHP 5.6 latest only, and iterate over various WP versions.
-  @require-wp-latest @require-php-5.6 @less-than-php-7.0
+  # Test on PHP 7.0 latest only, and iterate over various WP versions.
+  @require-wp-latest @require-php-7.0
```

Comment:
Hmm given the change in requirements we can't really test latest + PHP 7.0 anymore, so maybe what's needed here is test WP 6.2 / 4.2 / 3.9, all on PHP 5.6?

Originally added in 69f052d250b62597bdf27f898ea68ed9151feb16

Reply:
@swissspidy I think https://github.com/wp-cli/media-command/pull/181 addresses it!

Reply:
Thanks!